src/plans/create/steps/storage-map/utils.ts

```diff
@@ -74,27 +92,72 @@ const getStoragesUsedBySelectedVms = (
     ),
   );
 
-export const getSourceStorageLabels = (
+/**
+ * Creates a human-readable label for storage based on provider type
+ * Different providers have different naming conventions
+ */
+const getStorageMapLabel = (storage: InventoryStorage): string => {
```

Comment:
isnt it same exact switch for network as well?

Reply:
Almost. The Ovirt case is slightly different, but this is a good callout. They're similar enough to make a shared util, just with a single condition, so I'll do that.